tools/BuildAssets/targets/common.Build.props

```diff
@@ -18,38 +18,29 @@
 		<BuildWideDebugTrace Condition=" '$(DebugTrace)' == '' ">false</BuildWideDebugTrace>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
 	</PropertyGroup>
-
-	<!-- <PropertyGroup Condition=" '$(Configuration)' == 'Debug' AND '$(DelaySignForInternalsVisible)' == 'true' ">
-		There are certain projects that needs internals visible for test purpose
+	
+	<!--  AND ('$(CodeSign)' == 'true')
+	<PropertyGroup Condition=" ('$(IsBuildToolsProject)' == 'false' OR '$(IsBuildToolsProject)' == '') ">
```

Comment:
If you only ever set IsBuildToolsProject to true I would change this condition to `'$(IsBuildToolsProject)' != 'true'`

Reply:
@weshaggard sounds good

Reply:
Sorry I did not realized in the PR that the code was comment out.
There is no condition for IsBuildToolsProject == false anymore. The code that you see is commented out, so we are good.